File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
  let arr = [];
  let maxCandies = Math.max(...candies);
  for (let i = 0; i < candies.length; i++) {
    let candie = candies[i] + extraCandies;
    if (candie >= maxCandies) {
      arr.push(true);
    } else {
      arr.push(false);
    }
  }
  return arr;
};

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        max_candies = max(candies)
        results = []
        for i in range(len(candies)):
            results.append(candies[i] + extraCandies >= max_candies)
        return results